components/FrontPage/Combined/finalSection.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import TBox from "../textbox";
+import Btn from "../button";
+
+export default function FinalSection() {
+  return (
+    <div className="flex h-[400px] w-screen justify-center bg-slate-200 dark:bg-slate-800">
+      <div className="flex h-full max-w-[1200px] flex-col items-center justify-center gap-3">
```

Comment:
```suggestion
      <div className="flex h-full max-w-lg flex-col items-center justify-center gap-3">
```

We shouldn't be using so many absolute values.